chainHead: Allow single connection calls and enforce order of events (#137)
paritytech/json-rpc-interface-spec: src/api.md

Any missing parameter, or parameter with an invalid format, should result in a JSON-RPC error being returned, as described in the JSON-RPC specification.

> Any function returning an opaque subscription or operation ID ensures that the value is returned before any subsequent notifications are generated.

Reviewer: Nit: just felt a touch clearer

```suggestion
Any function returning an opaque subscription or operation ID ensures that this is returned before any related notifications are generated.
```
